airbyte-cdk/bulk/toolkits/load-csv/src/testFixtures/kotlin/io/airbyte/cdk/load/data/csv/CsvRowToAirbyteValue.kt

```diff
@@ -63,7 +64,8 @@ class CsvRowToAirbyteValue {
                     .fields()
                     .asSequence()
                     .map { entry ->
-                        entry.key to entry.value.toAirbyteValue(field.properties[entry.key]!!.type)
+                        val type = field.properties[entry.key]?.type ?: UnknownType("unknown")
+                        entry.key to entry.value.toAirbyteValue(type)
```

Comment:
csv needs (?) to explicitly pass through undeclared fields, so we just default to an unknown type